pkg/branding/brandedtext_test.go

```diff
@@ -0,0 +1,37 @@
+package branding
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/pkg/testutils/envisolator"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetBrandedProductName(t *testing.T) {
+	envIsolator := envisolator.NewEnvIsolator(t)
+
+	tests := map[string]struct {
+		productBrandingEnv string
+		brandedProductName string
+	}{
+		"RHACS branding": {
+			productBrandingEnv: "RHACS_BRANDING",
+			brandedProductName: productBrandingNameRHACS,
+		},
+		"Stackrox branding": {
+			productBrandingEnv: "STACKROX_BRANDING",
+			brandedProductName: productBrandingNameStackrox,
+		},
+		"Default setting": {
+			productBrandingEnv: "ROX_PRODUCT_BRANDING",
+			brandedProductName: productBrandingNameStackrox,
+		},
+	}
+	for name, tt := range tests {
+		t.Run(name, func(t *testing.T) {
+			envIsolator.Setenv(ProductBranding, tt.productBrandingEnv)
```

Comment:
I like that you use `"RHACS_BRANDING"` as a string and not reusing the const (which doesn't exist) in this test.
I would suggest to use `"ROX_PRODUCT_BRANDING"` instead of `ProductBranding` too.
```suggestion
			envIsolator.Setenv("ROX_PRODUCT_BRANDING", tt.productBrandingEnv)
```
The reason is that the test _sets the contract_. A change in the value of `ProductBranding` (i.e. expected environment variable name) will not break your _current_ test but it will break any usage that relies on the env variable.
When the test hardcodes the environment variable name and environment variable value, any change to them in the main code will break the test and will force the author of the change if they really want to do that. Hopefully, the author of the change would then grep the code for occurrences of `ROX_PRODUCT_BRANDING` to be able to make informed decision.

Reply:
Good point on hardcoding strings into tests, that's also an approach I used in the past. With that a comment I made earlier asking for clarification is also cleared up. I think I have a few other occurrences in tests where I did not hardcode strings this time around so I'll change those as well.